pkg/loadbalancer/loadbalancer.go

```diff
@@ -47,18 +48,19 @@ const (
 
 // ServiceFlags is the datapath representation of the service flags that can be
 // used (lb{4,6}_service.flags)
-type ServiceFlags uint8
+type ServiceFlags uint16
 
+const serviceFlagNone = 0
 const (
-	serviceFlagNone            = 0
-	serviceFlagExternalIPs     = 1
-	serviceFlagNodePort        = 2
-	serviceFlagLocalScope      = 4
-	serviceFlagHostPort        = 8
-	serviceFlagSessionAffinity = 16
-	serviceFlagLoadBalancer    = 32
-	serviceFlagRoutable        = 64
-	serviceFlagSourceRange     = 128
+	serviceFlagExternalIPs = 1 << iota
+	serviceFlagNodePort
+	serviceFlagLocalScope
+	serviceFlagHostPort
+	serviceFlagSessionAffinity
+	serviceFlagLoadBalancer
+	serviceFlagRoutable
+	serviceFlagSourceRange
+	serviceFlagLocalRedirect
```

Comment:
Nice refactor with the usage of `iota` :)

Reply:
Very tidy :-) Though I think this actually makes it even more difficult to pick up on the type of problem which I had pointed out earlier :thinking:

Reply:
Yeah, `iota` is nice but please add a comment on each line like.
``` golang
serviceFlagLocalRedirect // 1<<8
```

Reply:
I'd say if we're going to put the values here, better to explicitly declare them in code rather than comment (which could in theory be mismatched with the actual code)